.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+sudo: false
+node_js:
+  - 10
+install:
+  - npm i
+before_script:
+  - docker-compose up -d
```

Comment:
See https://docs.travis-ci.com/user/docker/, you need at least:

```yaml
services:
  - docker
```

Reply:
Fixed in f37831822127b31a8a00c0e85993f648037a2e1a